cypress/dockerfiles/interop/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM docker.io/cypress/included:13.13.3
```

Comment:
The container version should probably match the `cypress/package.json` version at `14.5.4` (from semver of `^14.5.3`).